tools/js-sdk-release-tools/src/common/utils.ts

```diff
@@ -298,6 +299,31 @@ export async function resolveOptions(typeSpecDirectory: string): Promise<Exclude
     return options
 }
 
+export function updateApiVersionInTspConfig(typeSpecDirectory: string, apiVersion?: string) {
```

Comment:
I feel like this naming is better
```suggestion
export function specifiyApiVersionToGenerateSDKByTypeSpec(typeSpecDirectory: string, apiVersion?: string) {
```